package.json

```diff
@@ -5,6 +5,7 @@
   "license": "Apache-2.0",
   "repository": "github:Netflix/eslint-plugin-x-element",
   "type": "module",
+  "main": "index.js",
```

Comment:
Is this just to be explicit? From the npm documentation on [main](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#main):

>If `main` is not set, it defaults to `index.js` in the package's root folder.